dialogue-core/src/main/java/com/palantir/dialogue/core/ConcurrencyLimitedChannel.java

```diff
@@ -41,26 +43,30 @@
     @Nullable
     private static final Void NO_CONTEXT = null;
 
-    private static final Ticker SYSTEM_NANOTIME = System::nanoTime;
-
-    private final Meter limitedMeter;
     private final LimitedChannel delegate;
-    private final Limiter<Void> limiter;
+    private final SimpleLimiter<Void> limiter;
+    private final Instrumentation instrumentation;
 
-    @VisibleForTesting
-    ConcurrencyLimitedChannel(LimitedChannel delegate, Limiter<Void> limiter, DialogueClientMetrics metrics) {
+    ConcurrencyLimitedChannel(LimitedChannel delegate, SimpleLimiter<Void> limiter, Instrumentation instrumentation) {
         this.delegate = new NeverThrowLimitedChannel(delegate);
-        this.limitedMeter = metrics.limited(getClass().getSimpleName());
+        this.instrumentation = instrumentation;
         this.limiter = limiter;
+        instrumentation.registerMaxGauge(limiter::getLimit);
+        instrumentation.registerUtilizationGauge(this::getUtilization);
     }
 
-    static ConcurrencyLimitedChannel create(LimitedChannel delegate, DialogueClientMetrics metrics) {
-        return new ConcurrencyLimitedChannel(
-                delegate, ConcurrencyLimitedChannel.createLimiter(SYSTEM_NANOTIME), metrics);
+    interface Instrumentation {
```

Comment:
why do we need this extra interface? can we just pass a `DialogueConcurrencyLimiterMetrics`?

Reply:
I had this originally, but it required passing in the string uri which was only used for instrumentation purposes (here's the commit where you can see this: https://github.com/palantir/dialogue/pull/443/commits/8a8139d09704d80974bd388df29df5b7e5df28ed).

This felt a bit gross, as the string uri could in theory be re-purposed for other stuff. Also felt odd to always be passing in the only possible tag value rather than just pre-combining them.